app/Http/Controllers/TaskController.php

```diff
@@ -56,7 +56,7 @@ public function store(Request $request) {
 
 		$task = $this->createTask($attributes);
 
-		return $task;
+		return Task::with('creator')->where('id', $task->id)->first();
```

Comment:
No tienes que hacer otro request a la BD. Usa el user que est&aacute; signed in, finalmente este es el m&eacute;todo que lo crea